_posts/2020-02-20-ddl.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "No Java Required: Configuring Sources and Sinks in SQL"
+date:  2020-02-20 12:00:00
+categories: news
+authors:
+- seth:
+  name: "Seth Wiesman"
+  twitter: "sjwiesman"
+
+
+excerpt: This post discusses the efforts of the Flink community as they relate to end to end applications with SQL in Apache Flink.
+---
+
+# Introduction
+
+The recent [Apache Flink 1.10 release](https://flink.apache.org/news/2020/02/11/release-1.10.0.html) includes many exciting features.
+In particular, it marks the end of the community's year-long effort to merge in the [Blink SQL contribution](https://flink.apache.org/news/2019/02/13/unified-batch-streaming-blink.html) from Alibaba.
+The reason the community chose to spend so much time on the contribution is that SQL works.
+It allows Flink to offer a truly unified interface over batch and streaming and makes stream processing accessible to a broad audience of developers and analysts.
+Best of all, Flink SQL is ANSI-SQL compliant, which means if you've ever used a database in the past, you already know it[^1]!
+
+A lot of work focused on improving runtime performance and progressively extending its coverage of the SQL standard.
+Flink now supports the full TPC-DS query set, reflecting the readiness of its SQL engine to address the needs of modern data warehouse-like workloads.
+
+As important as this work is, the community also strives to make these features generally accessible to the broadest audience possible.
+That is why the Flink community is excited in 1.10 to offer production-ready DDL syntax (e.g., `CREATE TABLE`, `DROP TABLE`) and a refactored catalog interface.
+
+# Accessing Your Data Where It Lives
+
+Flink does not store data at rest; it is a compute engine and requires other systems to consume input from and write its output.
+Those that have used Flink's `DataStream` API in the past will be familiar with connectors that allow for interacting with external systems. 
+Flink has a vast connector ecosystem that includes all major message queues, filesystems, and databases.
+
+<div class="alert alert-info">
+If your favorite system does not have a connector maintained in the central Apache Flink repository, check out the <a href="https://flink-packages.org">flink packages website</a>, which has a growing number of community-maintained components.
+</div>
+
+While these connectors are battle-tested and production-ready, they are written in Java and configured in code, which means they are not amenable to pure SQL or Table applications.
+Relational queries should execute against relational tables.
```

Comment:
I think I understand what you're trying to say, but would phrase it differently. A table can also be relational (following the relational data model) if it is not defined by a DDL statement.
How about something like this: "For a holistic SQL experience, not only queries need to be written in SQL, but also table definitions."?

Reply:
Yes, that's much better.